app/assets/javascripts/d3-wrapper-end.js

```diff
@@ -0,0 +1,3 @@
+return d3;
+  })();
+}, '0.1.0', {});
```

Comment:
I'm not 100% sure on this, but for convoy to pick it up this last line might need to be

```
}, '0.1.0', {
  requires: []
}); 
```